File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/DuplicateWalletTypeException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when registering a wallet type that has already been registered.
    /// </summary>
    public class DuplicateWalletTypeException : IndyException
    {
        /// <summary>
        /// Initializes a new DuplicateWalletTypeException.
        /// </summary>
        internal DuplicateWalletTypeException(string message) : base(message, (int)ErrorCode.WalletTypeAlreadyRegisteredError)
        {

        }
    }

}

File: wrappers/java/src/main/java/com/evernym/vdrtools/wallet/DuplicateWalletTypeException.java
package com.evernym.vdrtools.wallet;

import com.evernym.vdrtools.ErrorCode;
import com.evernym.vdrtools.IndyException;

/**
 * Exception thrown when attempting to register a custom wallet type that has already been registered.
 */
public class DuplicateWalletTypeException extends IndyException
{
	private static final long serialVersionUID = -5414881660233778407L;
	private final static String message = "A wallet type with the specified name has already been registered.";

	/**
	 * Initializes a new DuplicateWalletTypeException.
	 */
	public DuplicateWalletTypeException()
	{
		super(message, ErrorCode.WalletTypeAlreadyRegisteredError.value());
	}
}